doc/source/data/performance-tips.rst

```diff
@@ -187,5 +187,19 @@ To enable deterministic execution, set the preceding to True. This setting may d
 Monitoring your application
```

Comment:
In the future, let's probably move this section out as a separate page, because this is important.